telethon/tl/custom/dialog.py

```diff
@@ -1,6 +1,9 @@
 from . import Draft
 from .. import TLObject, types
+from ..functions.channels import LeaveChannelRequest
+from ..functions.messages import DeleteHistoryRequest
```

Comment:
Change this by editing the line above to be `from .. import TLObject, types, functions`.